### ISSUE TITLE
\Katana\SiteBuilder::build => $blogPostsFiles

### ISSUE THREAD

**Issue by Omranic** [themsaid/katana#14]
I've been using Katana on PHP7 normally, but when used on PHP 5.5.9 I've noticed that blog posts are compiled in an opposite order. When I debug the code, I find that file list is returned differently in both PHP versions:
```php
        $blogPostsFiles = array_filter($files, function ($file) {
            return str_contains($file->getRelativePath(), '_blog');
        });
```

### Compiled Order on PHP7
- 2016-04-17-a-week-of-laravel.md
- 2016-04-10-a-week-of-laravel.md

### Compiled Order on PHP5
- 2016-04-10-a-week-of-laravel.md
- 2016-04-17-a-week-of-laravel.md

### PHP Version
```
PHP 5.5.9-1ubuntu4.14 (cli) (built: Oct 28 2015 01:34:46)
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
    with Zend OPcache v7.0.3, Copyright (c) 1999-2014, by Zend Technologies
```

Suggestions?

**Comment by themsaid** [comment]
nice catch, can you PR a fix?

**Comment by Omranic** [comment]
Yeah, but I'm still not sure what's the real cause of this issue.
Do you think it's the change happened to `array_filter` function @ 5.6.0 ?

**Comment by caiquecastro** [comment]
I'm having a issue related to the post dates. I migrated my blog from jekyll. Now, I'm getting the blog posts in a totally random order. It's not being based on the file name. I'm trying to debug but it seems to be returning on this order from the filesystem. I've also started writing some unit tests but i'm not so experienced on unit tests.
